config/api.yml

```diff
@@ -1193,6 +1193,7 @@
     :description: Settings
     :identifier: ops_settings
     :options:
+    - :arbitrary_resource_path
     - :collection
```

Comment:
we need both, primarily to not confuse the two, a collection (i.e. exposed /api/...) which has special path requirement.